apps/cert-manager/OWNERS

```diff
@@ -1,8 +1,11 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-  - munnerz
+- munnerz
```

Comment:
So, one change that happened a lot was trying to make formatting consistent:
- deindent list items
- newline between approver/reviewer/emeritus_approver lists (and put them in that order)